build.gradle

```diff
@@ -3,23 +3,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 buildscript {
-    ext.support_libraries_version = '28.0.0'
-    ext.architecture_components_version = '1.1.1'
+    ext.support_libraries_version = '1.0.0'
+    ext.architecture_components_version = '2.0.0'
     ext.kotlin_version = '1.3.0'
     ext.moz_components_version = '0.43.0'
 
-    ext.robolectric_version = '4.0.1' // required for SDK 28.
+    ext.robolectric_version = '4.0.2' // required for SDK 28.
```

Comment:
nit: Add a comment for version bump reason - androidx?

Reply:
I should have when I did it.  I know there was a good reason for bumping this, but I honestly can't remember what it was.